dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -237,6 +245,39 @@ public static String getFileName(MultipartFile multipartFile)
         }
     }
 
+    /**
+     * This method will construct a List of BitstreamFormats out of a request.
+     * It will call the {@link Utils#getStringListFromRequest(HttpServletRequest)} method to retrieve a list of links
+     * out of the request.
+     * The method will iterate over this list of links and parse the links to retrieve the UUID from it.
+     * It will then look through all the DSpaceObjectServices to try and match this UUID to a DSpaceObject.
+     * If one is found, this DSpaceObject is added to the List of DSpaceObjects that we will return.
+     *
+     * @param request   The request out of which we'll create the List of BitstreamFormats
+     * @param context   The relevant DSpace context
+     * @return          The resulting list of BitstreamFormats that we parsed out of the request
+     */
+    public List<BitstreamFormat> constructBitstreamFormatList(HttpServletRequest request, Context context) {
+
+        return getStringListFromRequest(request).stream()
+                .map(link -> {
+                    if (link.endsWith("/")) {
+                        link = link.substring(0, link.length() - 1);
+                    }
+                    return link.substring(link.lastIndexOf('/') + 1);
+                })
+                .map(id -> {
+                    try {
+                        return bitstreamFormatService.find(context, parseInt(id));
+                    } catch (SQLException e) {
+                        log.error(e.getMessage(), e);
```

Comment:
Please log a more descriptive message here.  Keep in mind, `e.getMessage()` may be null....in which case this will say "Null".

We may also want to look more closely at the errors that could occur here. It seems like there are two likely errors...one is that `id` is NOT an Integer (and `parseInt` fails), and another is the BitstreamFormat cannot be found.